src/test/rustdoc-js/doc-alias.js

```diff
@@ -5,7 +5,7 @@ const QUERY = [
     'StructFieldItem',
     'StructMethodItem',
     'ImplTraitItem',
```

Comment:
`ImplTraitItem` should be removed from here as well now it's been removed from `doc-alias.rs`.

Reply:
That's why the search for it is returning nothing below. The goal was to check that nothing was returned.

Reply:
But that's the point of the test: it's supposed to not return anything, so we ensure that it doesn't.